yarn-project/end-to-end/src/e2e_epochs/epochs_l1_reorgs.test.ts

```diff
@@ -44,6 +48,20 @@ describe('e2e_epochs/epochs_l1_reorgs', () => {
     node = context.aztecNode;
     archiver = (node as AztecNodeService).getBlockSource() as Archiver;
     proverNode = context.proverNode!;
+
+    const account = privateKeyToAccount(generatePrivateKey());
+    l1ClientForMesssages = createExtendedL1Client(
+      [...test.l1Client.chain.rpcUrls.default.http],
+      account,
+      test.l1Client.chain,
+    );
+
+    const fundingTx = await test.l1Client.sendTransaction({
+      to: l1ClientForMesssages.account.address,
+      value: BigInt(1e16),
+    });
+
+    await test.l1Client.waitForTransactionReceipt({ hash: fundingTx });
```

Comment:
Don't we risk running into the same issue by reusing the sequencer's account here? We should be able to just pick up a different account from the anvil ones. See `getPrivateKeyFromIndex` in utils.